pkg/template/servicebroker/catalog.go

```diff
@@ -109,7 +117,11 @@ func serviceFromTemplate(template *templateapi.Template) *api.Service {
 	}
 }
 
+// Catalog returns our service catalog (one service per OpenShift template in
+// configured namespace(s)).
 func (b *Broker) Catalog() *api.Response {
+	glog.V(4).Infof("Template service broker: Catalog")
```

Comment:
maybe at least print the list of namespaces that it's fetching templates from.  this will help when someone says "but it's not retrieving all my templates"